includes/api/class-oembed-api.php

```diff
@@ -0,0 +1,137 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Class file names should be based on the class name with &quot;class-&quot; prepended. Expected class-oembed-api.php, but found phpcs-scan-8Nr6lp.php (*WordPress.Files.FileName.InvalidClassFileName*).